sdk/build.gradle

```diff
@@ -32,16 +32,14 @@ dependencies {
     implementation devDependencies.appCompat_v7
     implementation devDependencies.recyclerView_v7
     implementation devDependencies.constraintLayout
+    //TODO need to update last version
+    implementation 'com.github.kinecosystem.move-kin-android:appsTransfer:202f69f95e0447def46cb4fb4cdbab40a035d637'
```

Comment:
when it will be with the latest version and not a commit hash ?

Reply:
the coming days